docs/tutorial/getting-started.rst

```diff
@@ -112,6 +112,22 @@ iOS
   install the Xcode Command Line Tools. This can be done from the Preference
   panel within XCode itself.
 
+* For first time XCode users, make sure you've installed or updated to the latest XCode version.
+
+* You'll need to sign in to XCode with your Apple ID account - the same one you'd use for iTunes.
```

Comment:
This is needed for developing or for deploying?

Reply:
This is needed for developing - it is a prereq to be able to run the simulation on your computer or test on your phone after booting up the HelloWorld app.

Should I clarify with "To be able to run the simulator," ?

Reply:
Yes, if that's what you needed. To be clear, you needed a developer account to get the simulator up and running? Russell and I have never seen this, but our machines should be considered development waste dumps at this point and are so far from the normal beginner experience we can see Saturn.

If you had to do this, then definitely add the documentation.